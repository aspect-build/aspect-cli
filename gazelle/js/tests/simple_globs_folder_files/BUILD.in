# gazelle:js_test_files mocks **/__mocks__/**
# gazelle:js_test_files **/*.test.ts
