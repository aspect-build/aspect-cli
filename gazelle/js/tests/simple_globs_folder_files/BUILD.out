load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

ts_project(
    name = "simple_globs",
    srcs = ["srcs/main.ts"],
)

ts_project(
    name = "simple_globs_tests",
    testonly = True,
    srcs = ["tests/main.ts"],
    deps = [":simple_globs"],
)
